tests/test_pysat_cardinality.py

```diff
@@ -6,6 +7,7 @@
 from cpmpy.solvers.pysat import CPM_pysat
 from cpmpy.transformations.linearize import only_positive_coefficients
 
+pblib_available = importlib.util.find_spec("pypblib") is not None
```

Comment:
Can you add a comment here?
Just for future reference, something saying: "pypblib is required for encoding cardinality constraints to clauses in the pysat interface" or smt (same on the other tests)